src/plugins/dashboard/public/application/listing/dashboard_listing.js

```diff
@@ -201,16 +216,17 @@ export class DashboardListing extends React.Component {
         render: (updatedAt) => updatedAt && moment(updatedAt).format(dateFormat),
       },
     ];
-    return tableColumns;
   }
 }
 
 DashboardListing.propTypes = {
-  createItem: PropTypes.func.isRequired,
+  createItem: PropTypes.func,
+  dashboardProviders: PropTypes.object,
   findItems: PropTypes.func.isRequired,
   deleteItems: PropTypes.func.isRequired,
   editItem: PropTypes.func.isRequired,
-  getViewUrl: PropTypes.func.isRequired,
```

Comment:
@pjfitzgibbons do you know if this DashboardListing is strictly used internally and cannot be used externally?

This appears to be exported and I think in main it will be fine to remove and but if for 2.x to add a new prop by removing it will appear to be a breaking change on the interface. Not that I am able to find any thing utilizing the DashboardListing interface besides the dashboard plugin.